web/css/themes/makeITup/catro_bootstrap.less

```diff
@@ -0,0 +1,323 @@
+      color: white;
+      margin: 0;
+    }
+
+    .navbar-toggle
+    {
+      margin-top: 0;
+      margin-bottom: 0;
+    }
+  }
+
+  #bs-navbar-collapse
+  {
+    max-height: none;
+
+    .navbar-small
+    {
+      padding-top: 15px;
+      padding-bottom: 15px;
+    }
+
+    .navbar-nav
+    {
+      margin: 0;
+
+      .navbar-form
+      {
+        margin: 0;
+      }
+
+      li
+      {
+        a
+        {
+          display: flex;
+          visibility: visible;
+          align-items: center;
+          h4
+          {
+            margin-left: auto;
+          }
+        }
+      }
+    }
+  }
+}
+
+//################## Footer ####################
+@padding: 15px;
+@margin-adjustment: -5px;
+
+footer
+{
+  background-color: #E1E2E1;
+  padding: @padding 0;
+  ul
+  {
+    margin-bottom: 0;
+    padding: 0;
+    list-style: none;
+
+    li
+    {
+      display: inline-block;
+
+      a
+      {
+        color: inherit;
+        font-weight: 500;
+        font-size: 12px;
+        text-transform: uppercase;
+        text-decoration: none;
+
+        position: relative;
+        padding: @padding;
+        display: block;
+
+        border-radius: 3px;
+        &:hover
+        {
+          text-decoration: none;
+        }
+      }
+
+      .btn
+      {
+        margin: 0;
+      }
+    }
+
+    &.list-horizontal
+    {
+      &:first-child a
+      {
+        padding-left: 0;
+      }
+
+      &:last-child a
+      {
+        padding-right: 0;
+      }
+    }
+
+    &.list-vertical
+    {
+      li
+      {
+        margin-left: @margin-adjustment;
+        margin-right: @margin-adjustment;
+        display: block;
+
+        a
+        {
+          padding: 5px;
+        }
+      }
+    }
+  }
+
+  .pull-center
+  {
+    display: inline-block;
+    float: none;
+  }
+  h4
+  {
+    font-weight: 500;
+    color: #000000;
+  }
+}
+//################## Footer End ####################
+
+
+
+// can/should be used for uniform width of page containers
+.content
+{
+  height: 100%;
+}
+.navbar-brand #logo
+{
+  margin-top: -25px;
+}
+
+@navbar-collapse-width:    ~"only screen and (max-width: 823px)";
+
+@media @navbar-collapse-width
+{
+  #logo
+  {
+    height: 63px;
+  }
+}
+
+@media screen and (min-width: @screen-sm)
+{
+  .navbar-small
+  {
+    display: none;
+  }
+
+  .navbar-big
+  {
+    display: inherit;
+  }
+
+  #bs-navbar-collapse
+  {
+    .navbar-nav
+    {
+      align-items: center;
+      display: flex;
+      padding-top: 25px;
+
+      li
+      {
+        padding: 5px;
+
+        a
+        {
+          padding-top: 0;
+          padding-bottom: 0;
+        }
+
+        span
+        {
+          margin-right: 10px;
+        }
+
+        ul
+        {
+          padding: 5px;
+          min-width: 210px;
+
+          a
+          {
+            padding-top: 0;
+            padding-bottom: 0;
+          }
+
+          span
+          {
+            margin-right: 10px;
+          }
+        }
+      }
+    }
+  }
+}
```

Comment:
I believe there is a way to import the base theme and then overwrite the needed things.
This looks like you copied a whole file over which is something you should never do. Just overwrite the things that should be different.

Reply:
As we discussed, this is really an issue. I just ran into the situation that I would have to copy a change into by now 7 different files.

That's not how we initially intended the `catro_bootstrap.less` to be used.

We will solve this issue inside this pr.